components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m

```diff
@@ -0,0 +1,215 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCActionSheetItemTableViewCell.h"
+
+#import "MaterialTypography.h"
+
+static const CGFloat kImageAlpha = 0.54f;
+static const CGFloat kCellLabelAlpha = 0.87f;
+static const CGFloat kLeadingPadding = 16.f;
+static const CGFloat kActionItemTitleVerticalPadding = 18.f;
+
+@interface MDCActionSheetItemTableViewCell ()
+@end
+
+
+@implementation MDCActionSheetItemTableViewCell {
+  MDCActionSheetAction *_itemAction;
+  UILabel *_textLabel;
+  UIImageView *_imageView;
+  NSLayoutConstraint *_leadingTitleConstraint;
+  MDCInkTouchController *_inkTouchController;
+  NSLayoutConstraint *_widthConstraint;
+}
+
+@synthesize mdc_adjustsFontForContentSizeCategory;
```

Comment:
Don't synthesize if you are providing an implementation.

Reply:
He's only implementing the setter, so this synthesize creates an ivar for him.